secretstores/metadata.go

```diff
@@ -20,3 +20,10 @@ const DefaultSecretRefKeyName = "_value"
 type Metadata struct {
 	Properties map[string]string `json:"properties,omitempty"`
 }
+
+type ParameterStoreMetaData struct {
```

Comment:
If this is specific to tencent/ssm, please move it to the tencent/ssm/ssm.go file

Reply:
This is redundant. My original idea is that the current secretid and secretkey, token, and region names are not standardized, so I wanted to unify them.

Can we make some sematic conversions? Otherwise the configuration names of each component are not uniform, and the experience is very poor.

Later, I want to mention such issues and PRs separately

Reply:
Agree. Thanks.